src/Microsoft.TemplateEngine.Core/Operations/Conditional.cs

```diff
@@ -318,32 +306,28 @@ public int HandleMatch(IProcessorState processor, int bufferLength, ref int curr
                     }
 
                     _current.BranchTaken = true;
-                    processor.WhitespaceHandler(ref bufferLength, ref currentBufferPosition, wholeLine: _definition._wholeLine, trim: _definition._trimWhitespace);
+                    processor.WhitespaceHandler(ref bufferLength, ref currentBufferPosition, _definition.WholeLine, _definition.TrimWhitespace);
                     return 0;
                 }
-                else
-                {
-                    Debug.Assert(true, "Unknown token index: " + token);
-                    return 0;   // TODO: revisit. Not sure what's best here.
-                }
+
+                Debug.Assert(true, "Unknown token index: " + token);
+                return 0;   // TODO: revisit. Not sure what's best here.
             }
 
             // moves the buffer to the next token at the same level.
             // Returns false if no end token can be found at the same level.
             //      this is probably indicative of a template authoring problem, or possibly a buffer problem.
-            private bool SkipToMatchingEndif(IProcessorState processor, ref int bufferLength, ref int currentBufferPosition, ref int token)
+            private void SkipToMatchingEndif(IProcessorState processor, ref int bufferLength, ref int currentBufferPosition, ref int token)
```

Comment:
The return value is not used where this is called from, so I can see the case for changing it from bool to void. But the return value indicates a potential token-balancing problem which could (should?) be checked for. Without it, there's no way to check for the problem.